crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml

```diff
@@ -0,0 +1,55 @@
+<odoo>
+    <data>
+        <template id="myaccount_crowdfunding_view_template" name="MyAccount crowdfunding template">
+            <t t-call="website.layout">
+
+                <h2 class="blue my-2 center">Personal data</h2>
+                <section class="center">
+                    <t t-call="crowdfunding_compassion.myaccount_coordinates"/>
+                </section>
+
+                <h2 class="blue my-2 center">Your own projects</h2>
+                <section class="projects no-padding-top center">
+                    <t t-set="projects" t-value="owned_projects"/>
+                    <t t-call="crowdfunding_compassion.myaccount_project_board"/>
+                </section>
+
+                <h2 class="blue my-2 center">Projects you participate to</h2>
+                <section class="projects no-padding-top center">
+                    <t t-set="participations" t-value="participating_projects"/>
+                    <t t-call="crowdfunding_compassion.myaccount_participation_board"/>
+                </section>
+
+                <h2 class="blue my-2 center">Donations</h2>
+                <section class="center">
+                    <t t-set="donations" t-value="donations"/>
+                    <t t-call="crowdfunding_compassion.myaccount_donation_board"/>
+                </section>
+            </t>
+        </template>
+
+        <record id="myaccount_crowdfunding_view" model="ir.ui.view">
+            <field name="name">MyAccount view</field>
+            <field name="type">qweb</field>
+            <field name="key">crowdfunding.myaccount</field>
+            <field name="arch" type="xml">
+                <t t-call="crowdfunding_compassion.myaccount_crowdfunding_view_template"/>
+            </field>
+        </record>
+
+        <record id="myaccount_crowdfunding_page" model="website.page">
+            <field name="url">/my_account</field>
+            <field name="website_indexed" eval="True"/>
+            <field name="website_id" eval="ref('crowdfunding_compassion.crowdfunding_website')"/>
+            <field name="website_published">True</field>
+            <field name="name">MyAccount</field>
+            <field name="type">qweb</field>
+        </record>
+
+        <template id="my_account_crowdfunding_link" name="Link to MyAccount page" inherit_id="portal.frontend_layout">
```

Comment:
I think this mean you will change the regular my account page as well. Don't think it's the right way to do it. I think we have a separate navbar for the crowdfunding website and we should have a user icon here with the link to the /my_account, and not touch the default my home menu for the general website.

Reply:
Yes I did that when I was learning the system and it's not useful anymore so I will delete it